docs/data/introduction/installation/installation.md

```diff
@@ -0,0 +1,187 @@
+# Installation
+
+<p class="description">Install the necessary packages to start building with MUI X components.</p>
+
+MUI X packages are available through the (free, MIT-licensed) Community Plan, or the (commercially-licensed) Pro and Premium Plans.
+See [Licensing](/x/introduction/licensing/) for complete details.
+
+Note that you only need to install the packages corresponding to the components you're using—e.g. Data Grid users do not need to install Date and Time Pickers.
```

Comment:
Hmm yeah I couldn't decide if this was callout-worthy or not. 😅

Reply:
Probably not, and reading it again, I wonder if it isn't stating the blindingly obvious?

Reply:
Haha, I went through the same thought process myself. Ultimately I think it's worth spelling out because MUI X packages follow a different pattern than the Core libraries—so if you're coming from Material UI for instance, you might assume that there's just one package called something like `@mui/x` to install.